packages/ripple-nuxt-tide/lib/core/tide.js

```diff
@@ -420,19 +420,18 @@ export const tide = (axios, site, config) => ({
     return pageData
   },
 
-  getPreviewPage: async function (contentType, uuid, revisionId, section, params, headersConfig) {
-    if (revisionId === 'latest') {
-      params.resourceVersion = 'rel:working-copy'
-    } else {
-      params.resourceVersion = `id:${revisionId}`
-    }
+  getPageByPreviewLink: async function (path, section) {
```

Comment:
Hey @alan-cole should we move this into preview module packages/ripple-nuxt-tide/modules/preview/lib/helpers.js ?

Reply:
Yep, will need to import jsonapiParse and tide, and export this function for use in middleware. One question, are you fine with isPreviewPath being in core, or would you like that in the preview module too?

Reply:
@alan-cole Thanks. Let's leave `isPreviewPath ` in core. As it need to be used by core code.